assets/src/application/ui/layout/espressoTable/style.scss

```diff
@@ -28,6 +30,10 @@
 	&__is-scrollable .ee-rspnsv-table__inner-wrapper {
 		overflow-x: scroll !important;
 		overflow-y: visible;
+
+		@include max782px {
+			overflow-x: unset !important;
```

Comment:
fix for the undesired scroll on mobile:

![image](https://user-images.githubusercontent.com/1477453/83554170-04b3a600-a515-11ea-8a6e-81aea9373999.png)

Reply:
ya that looks a **LOT** better 🏅 

I have a few requests/suggestions based on the above image:

- decrease the space (margin or padding) on either side of the TAM table. Maybe half of what there would be good. Don't need that much white space on mobile when real estate is so expensive.

- button height is fine, but they don't need to be that wide, so maybe make them square, push them all the way to the right, and then use all of the gained space for the ticket info.

- for the ticket info, let's try the following:

	- move the sale date with the coloured status background to the far left of the cell, and increase the height to 100% of the cell interior (ie: keep the same spacing that exists now between the date block and the cell outline)

	- don't know if this is possible as is, (might have to add some additional markup like span tags) but put the month, day, and year portions for the dates on their own lines and center align them 

end result for layout should look kinda like:

```
**********
*   Jun  *					ID: 117	   * * * *  
*   03   *		Test Test Test Test	   * btn *  
*  2020  *					  $ 115	   * * * *  
**********
```

- finally, let's make the ID text a little bit lighter in color, and then maybe bold the ticket title

Reply:
relation button adjusted:

![image](https://user-images.githubusercontent.com/1477453/83641580-69730d00-a5b6-11ea-8887-f9c3d2016ad1.png)

Reply:
hope this is close to what was expected:

![image](https://user-images.githubusercontent.com/1477453/83648904-46009000-a5bf-11ea-9839-396f34122253.png)
***
- first, I've applied overflow hidden and ellipsis for name because multi line title would have created problems with space occupied by price
- I positioned price absolutely because it was tricky to arrange those 3 elements as expected

Reply:
it's looking better... don't like the offset text alignments though...

let's just left align all of the ticket info so that all of that text is consistent and we don't have to deal with absolute positioning, which can cause unexpected issues